Jenkinsfile

```diff
@@ -103,9 +103,8 @@ pipeline{
 						}
 					}
 				}
-				stage ('Worker build SGX') {
+				stage ('Worker build SGX & VFIO') {
 					agent { node { label 'bionic-sgx' } }
```

Comment:
The better thing to do is add a new label `bionic-nvdidia-vfio` and assign it to the same node. And then add a new block that needs to run on that label.

Reply:
Done! I've renamed the node `bionic-sgx` into `bionic-baremetal`, and I've updated the list of labels from `bionic-sgx` to `bionic-sgx bionic-vfio`.